components/AlertsList.vue

```diff
@@ -1,15 +1,16 @@
 <template>
-  <v-container class="alerts-list">
+  <v-container class="pb-sm-0 ma-sm-0">
```

Comment:
What are the `sm` for?

Usually the classes are like `pb-0 ma-0`. You can experiment and see visuals in the playground: https://v15.vuetifyjs.com/en/framework/spacing#playground

Reply:
The sm is so it only applies to mobile screens.

Reply:
🤦🏻‍♀️ We're on vuetify 2.5.... that's awesome

I'd been using 1.5 docs because I misread the version `"@nuxtjs/vuetify": "^1.11.3",` but... 🤦🏻 

No wonder my old syntax didn't work 😆